packages/components/ovh-at-internet/src/ovh-at-internet.ts

```diff
@@ -179,16 +181,19 @@ export default class OvhAtInternet extends OvhAtInternetConfig {
           updateVisitorId(this.tag.clientSideUserId.get());
         }
       }
+      // Init mix commander
+      initMixCommander(this.getGenericTrackingData({ name: '', level2: '' }));
```

Comment:
We can do it in `init` function as 'Mix-commander' is independent of "Piano-analytics"